products/memgraph/v1.2.0/quick-start.md

```diff
@@ -0,0 +1,306 @@
+Finally, please be aware of and understand the implications of
+[telemetry](#telemetry) as used by the Memgraph team.
+
+### Installation
+
+Memgraph is available to [download as a binary](https://memgraph.com/download/)
+in the following formats:
+
+  * [Debian package for Debian 9 (Stretch)](#debian-installation)
+  * [RPM package for CentOS 7](#RPM-installation)
+  * [Docker image](#docker-installation)
+
+It is important to note that newer versions of Memgraph are currently not
+backward compatible with older versions. This is mainly noticeable by
+being unable to load storage snapshots between different versions.
+
+#### Debian Package Installation {#debian-installation}
+
+After downloading Memgraph as a Debian package, install it by running the
+following:
+
+```plaintext
+dpkg -i /path/to/memgraph_<version>.deb
+```
+
+You could get errors while installing the package with the above command if you
+don't have all of Memgraph's dependencies installed. The issues mostly look
+like the following:
+
+```plaintext
+dpkg: error processing package memgraph (--install):
+ dependency problems - leaving unconfigured
+Errors were encountered while processing:
+ memgraph
+```
+
+To install missing dependencies and finish the installation of the Memgraph
+package, just issue the following command:
+
+```plaintext
+apt-get install -f
+```
+
+The above command will install all missing dependencies and will finish
+configuring the Memgraph package.
+
+On successful installation, Memgraph should already be running. To
+confirm it, you can start it explicitly as follows:
+
+
+```plaintext
+systemctl start memgraph
+```
+
+To verify that Memgraph is running, run the following:
+
+```plaintext
+journalctl --unit memgraph
+```
+
+If successful, you should receive an output similar to the following:
+
+```plaintext
+Nov 23 13:40:13 hostname memgraph[14654]: Starting 8 BoltS workers
+Nov 23 13:40:13 hostname memgraph[14654]: BoltS server is fully armed and operational
+Nov 23 13:40:13 hostname memgraph[14654]: BoltS listening on 0.0.0.0 at 7687
+```
+
+At this point, Memgraph is ready for you to [submit queries](#querying).
+
+##### Configuration
+The Memgraph configuration is available in `/etc/memgraph/memgraph.conf`.
+
+
+#### RPM Package Installation {#RPM-installation}
+
+After downloading the RPM package of Memgraph, you can install it by
+issuing the following command:
+
+```plaintext
+yum --nogpgcheck localinstall /path/to/memgraph-<version>.rpm
+```
+
+After successful installation, Memgraph can be started as a
+service using the following command:
+
+```plaintext
+systemctl start memgraph
+```
+
+To verify that Memgraph is running, run the following command:
+
+```plaintext
+journalctl --unit memgraph
+```
+
+If successful, the you should receive an output similar to the following:
+
+```plaintext
+Nov 23 13:40:13 hostname memgraph[14654]: Starting 8 BoltS workers
```

Comment:
Missing the connection information output but not critical

Reply:
done